.github/workflows/accessibility.yml

```diff
@@ -0,0 +1,47 @@
+name: accessibility
```

Comment:
Most of the code is copied and pasted from the main [cypress.yml](https://github.com/canonical-web-and-design/maas-ui/master/.github/workflows/cypress.yml)

This isn't something introduced by this PR though. I created a separate issue for reducing duplication in our actions using action composition.

Reply:
https://github.com/canonical-web-and-design/maas-ui/issues/3905